### PR TITLE
GSE proposal for updating Get school experience page

### PR DESCRIPTION
**Pull request by MylesJarvis**
### Trello card
N/A

### Context
As part of our work in GSE, we've been working on a new draft for the GSE page. This is an initial tightening up of the language on the page. This makes the GSE service itself a bit more visible.

We think there's probably more ambitious work that we can do with the Website team, but this is a start.

### Guidance to review
@Tomcowan1 to review and provide feedback in the first instance (if possible!)

### CODE REVIEW

File: app/views/content/get-school-experience.md [hunk 1]
```diff
@@ -21,39 +21,34 @@ keywords:
...
+## Maths or physics teaching internships
 
-Be flexible with your availability if possible and persevere with your search.
+You may be eligible for a [maths or physics teaching internship](/teaching-internship-providers) earning £300 a week if you're doing a undergraduate degree in science, technology, engineering or maths.
 
-## Maths or physics teaching internships
+## Attend teaching events
 
-You may be eligible for a [maths or physics teaching internship earning £300 a week if you're doing a undergraduate degree in science, technology, engineering or maths](/teaching-internship-providers).
+[Teaching events](/events) can also give you an insight into teaching. You can find Train to Teach events, online ‘Q and A’ sessions as well as university events.
```
Comment (gemmadallmandfe) [inline app/views/content/get-school-experience.md:46]:
```suggestion
[Teaching events](/events) can also give you an insight into teaching. You can choose from our Train to Teach events, online Q&A sessions and events run by training providers.
```

File: app/views/content/get-school-experience.md [hunk 2]
```diff
@@ -21,39 +21,34 @@ keywords:
...
+* find out how specific subjects are taught
+* speak to teachers and interact with pupils
+* learn more about teacher training - including the application and interview process
 
-To find school experience you can:
+## Find school experience
 
-* [find a school offering experience through GOV.UK](https://schoolexperience.education.gov.uk/)
-* approach schools yourself
+Use the [Get school experience service](https://schoolexperience.education.gov.uk/) to search for and request experience in England.
```
Comment (gemmadallmandfe) [inline app/views/content/get-school-experience.md:36]:
```suggestion
Use our [Get school experience service](https://schoolexperience.education.gov.uk/) to search for and request experience in England.
```

**Comment by gemmadallmandfe** [comment]
Couple of suggestions from me too @MylesJarvis - mostly re how we refer to events elsewhere

We'll be changing the phrase 'Teaching events' at some point very soon but will come back to that one!

**Comment by Tomcowan1** [comment]
Sorry @MylesJarvis just one more thing. The maths internship page you're linking to might be a bit of a dead end (internships haven't opened for this academic year yet - will most likely open in the NY). We'll update the page once it's all been agreed, but for now you might want to take that bit out and add it back in once the internships are live?

**Comment by MylesJarvis** [comment]
@Tomcowan1 can I get a final review on this once the new review app has deployed?

**Review by Tomcowan1 — APPROVED** [review]

**Review by gemmadallmandfe — APPROVED** [review]
Looks good to me - happy for you to merge @MylesJarvis 

One minor comment - we will get flagged re accessibility of links as we have multiple links going to the same destination which use different link text (Get school experience vs Get school experience service). Ideally this need updating!

**Pull request merged**